Common/GPU/Vulkan/VulkanRenderManager.cpp

```diff
@@ -562,14 +517,12 @@ void VulkanRenderManager::BeginFrame(bool enableProfiling, bool enableLogProfile
 	FrameData &frameData = frameData_[curFrame];
 
 	// Make sure the very last command buffer from the frame before the previous has been fully executed.
-	if (useThread_) {
-		std::unique_lock<std::mutex> lock(frameData.push_mutex);
-		while (!frameData.readyForFence) {
-			VLOG("PUSH: Waiting for frame[%d].readyForFence = 1", curFrame);
-			frameData.push_condVar.wait(lock);
-		}
-		frameData.readyForFence = false;
+	std::unique_lock<std::mutex> lock(frameData.push_mutex);
```

Comment:
Hm, it's looks okay, but this holds the push mutex lock a lot longer than before.  Intentional?

-[Unknown]

Reply:
Thanks, I caught myself from doing this in another place but somehow missed this one.